packages/go_router_builder/README.md

```diff
@@ -93,71 +101,83 @@ class HomeRoute extends GoRouteData {
 
 The tree of routes is defined as an attribute on each of the top-level routes:
 
+<?code-excerpt "example/lib/main.dart (RouteTree)"?>
 ```dart
 @TypedGoRoute<HomeRoute>(
   path: '/',
   routes: <TypedGoRoute<GoRouteData>>[
     TypedGoRoute<FamilyRoute>(
-      path: 'family/:familyId',
-    )
+      path: 'family/:fid',
+// ···
   ],
 )
 class HomeRoute extends GoRouteData {
   const HomeRoute();
 
   @override
-  Widget build(BuildContext context, GoRouterState state) => HomeScreen(families: familyData);
+  Widget build(BuildContext context, GoRouterState state) => const HomeScreen();
 }
 
-@TypedGoRoute<LoginRoute>(path: '/login')
-class LoginRoute extends GoRouteData {...}
+@TypedGoRoute<LoginRoute>(
+  path: '/login',
+)
+class LoginRoute extends GoRouteData {
+// ···
+}
 ```
 
 ## `GoRouter` initialization
 
 The code generator aggregates all top-level routes into a single list called
 `$appRoutes` for use in initializing the `GoRouter` instance:
 
+<?code-excerpt "example/lib/simple_example.dart (Initialization)"?>
 ```dart
-final _router = GoRouter(routes: $appRoutes);
+final GoRouter _router = GoRouter(routes: $appRoutes);
 ```
 
 ## Error builder
 
 One can use typed routes to provide an error builder as well:
 
+<?code-excerpt "example/lib/readme_excerpts.dart (ErrorBuilder)"?>
 ```dart
 class ErrorRoute extends GoRouteData {
   ErrorRoute({required this.error});
   final Exception error;
 
   @override
-  Widget build(BuildContext context, GoRouterState state) => ErrorScreen(error: error);
+  Widget build(BuildContext context, GoRouterState state) =>
+      ErrorScreen(error: error);
 }
 ```
 
 With this in place, you can provide the `errorBuilder` parameter like so:
 
+<?code-excerpt "example/lib/readme_excerpts.dart (ErrorBuilderParameter)"?>
 ```dart
-final _router = GoRouter(
-  routes: $appRoutes,
-  errorBuilder: (c, s) => ErrorRoute(s.error!).build(c),
-);
+  final GoRouter _router = GoRouter(
+    errorBuilder: (BuildContext c, GoRouterState s) =>
+        ErrorRoute(error: s.error!).build(c, s),
+// ···
+  );
 ```
 
 ## Navigation
 
 Navigate using the `go` or `push` methods provided by the code generator:
 
+<?code-excerpt "example/lib/main.dart (Navigation)"?>
 ```dart
-void _tap() => PersonRoute(fid: 'f2', pid: 'p1').go(context);
+onTap: () => PersonRoute(family.id, p.id).go(context),
```

Comment:
I'm not familiar with the go_router code; are there both named *and* positional versions of the constructors? I see one below that still have the `pid:`.

If so, it seems like we should be using the named versions, as they are safer?